ansible/liquid.yml

```diff
@@ -12,10 +12,10 @@
     - network
     - discover
     - liquid-core
-    - node6
     - web-ui
-    - hoover
-    - hypothesis
-    - dokuwiki
-    - matrix
-    - davros
+    - {tags: ['apps'], role: node6}
```

Comment:
the `web-ui` role installs `ui-templates` that runs `install.py` that does a `npm run` or something.
That means `initiallize.d/70-install-templates` is gonna fail without node6.